scripts/docker/zeppelin/build-from-distribution/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
it looks like it is almost the same of `scripts/docker/zeppelin/bin/Dockerfile`, is it possible to reuse that ?

Reply:
Actually, the dockerfile is different from the `scripts/docker/zeppelin/bin/Dockerfile` because it needs to build the docker image from the distribution tar.gz. But the properties file about log4j can be reused (copy from `scripts/docker/zeppelin/bin/` before the image building).